com.github.diegopvlk.Dosage.json

```diff
@@ -0,0 +1,54 @@
+{
+    "app-id" : "com.github.diegopvlk.Dosage",
```

Comment:
Should be io.github, see https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
Looks like you need to make a bunch of upstream changes too, stuff there assumes the id `com.github`